### DISCUSSION TITLE
Doubt about symbol meaning of GraphConv

### DISCUSSION THREAD

**Discussion by DogitoErgoSum** [pyg-team/pytorch_geometric#4473] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/51344466/163208694-22382f8d-ac32-4f97-a5c4-5a3150a00a9c.png)
![image](https://user-images.githubusercontent.com/51344466/163208767-15d2753c-8e2d-40aa-a19a-12ae186c5770.png)

Is this symbol a weight matrix? it seems yes from the paper but in others convs its used a "W" to denote a weight matrix.

**Comment by rusty1s** [comment, marked as answer]
Thanks for reporting. We use Theta and W interchangeably, and could definitely consider to unify this notation. Let me know if you want to fix this :)

**Reply by rusty1s** [reply]
Partially fixed in https://github.com/pyg-team/pytorch_geometric/pull/4495.
